README.md

```diff
@@ -80,32 +95,12 @@ in the `dependencies` section of the `package.json` file, not in the
 even if the app that uses the addon does not use ember-test-selectors itself.
 
 
-Configuration
-------------------------------------------------------------------------------
-
-You can override when the `data-test-*` attributes should be stripped from the
-build by modifying your `ember-cli-build.js` file:
-
-```js
-var app = new EmberApp({
-  'ember-test-selectors': {
-    strip: false
-  }
-});
-```
-
-`strip` accepts a `Boolean` value and defaults to `!app.tests`, which means
-that the attributes will be stripped for production builds, unless the build
-was triggered by `ember test`. That means that if you use
-`ember test --environment=production` the test selectors will still work, but
-for `ember build -prod` they will be stripped out.
-
 License
 ------------------------------------------------------------------------------
 
 ember-test-selectors is developed by and &copy;
 [Mainmatter GmbH](http://mainmatter.com) and contributors. It is released under the
-[MIT License](https://github.com/mainmatter/ember-simple-auth/blob/master/LICENSE).
+[MIT License](https://github.com/mainmatter/ember-test-selectors/blob/master/LICENSE).
```

Comment:
so it looks like you deleted the top-level LICENCE file but we're linking to it here? 🤔 maybe leave the top level one?

Reply:
I deleted the low-level LICENSE in `ember-test-selectors/ember-test-selectors` and now all the READMEs are pointing to the top level one.